templates/flows/flow_pdf_export_next.haml

```diff
@@ -0,0 +1,282 @@
+{% extends "smartmin/base.html" %}
```

Comment:
@srugano - It's missing to remove this template file since we are going to use `react-to-print` for the new flow editor for the PDF export feature